src/Module/Circle.php

```diff
@@ -141,13 +141,15 @@ public function ajaxPost()
 
 	protected function content(array $request = []): string
 	{
-		$change = false;
+		$change   = false;
+		$relation = $request['rel'] ?? '';
 
 		if (!DI::userSession()->getLocalUserId()) {
 			throw new \Friendica\Network\HTTPException\ForbiddenException();
 		}
 
 		DI::page()['aside'] = Model\Circle::sidebarWidget('contact', 'circle', 'extended', ((DI::args()->getArgc() > 1) ? DI::args()->getArgv()[1] : 'everyone'));
+		DI::page()['aside'] .= Widget::contactRels($_SERVER['REQUEST_URI'], $relation);
```

Comment:
This is available as `$this->server` now
```suggestion
		DI::page()['aside'] .= Widget::contactRels($this->server['REQUEST_URI'], $relation);
```

Reply:
Did so now